src/styles/Header/header.scss

```
@@ -0,0 +1,65 @@
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  font-style: normal;
```

```suggestion
  font-style: normal;
```

it's default value so no need to set it